packages/ui-react/src/components/Account/Account.tsx

```diff
@@ -136,23 +136,29 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
       .forEach((error) => {
         switch (error.trim()) {
           case 'Invalid param email':
+            formErrors.form = t('account.errors.validation_error');
```

Comment:
This and the other references to `t('account.errors.validation_error')` are not correct yet, it will now result in:

<img width="60%" src="https://github.com/Videodock/ott-web-app/assets/48496458/9b139122-0620-46dc-8c7c-ecbeb0abf14d">

Reply:
I accidentally discarded this change. I'll add it again. Thanks